src/platform.ts

```diff
@@ -309,9 +315,21 @@ export class PlatformInformation {
                 // Oracle Linux is binary compatible with CentOS
                 return centos_7;
             case 'fedora':
-                return fedora_23;
+                if (distributionVersion === "23") {
+                    return fedora_23;
+                } else if (distributionVersion === "24") {
+                    return fedora_24;
+                }
+                break;
+
             case 'opensuse':
-                return opensuse_13_2;
+                if (distributionVersion.startsWith("13.")) {
+                    return opensuse_13_2;
+                } else if (distributionVersion.startsWith("42.")) {
```

Comment:
I think we should fully qualify the version 13.2 and 42.1, since coreclr seems to exactly match the version.